modules/cnf-du-updating-crio.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-crio_{context}"]
+
+= Updating CRI-O
+
+In support of this feature, CRI-O supports new configuration settings. These are applied by the MachineConfig specified above.
```

Comment:
Maybe "To support workload partitioning, CRI-O..."

My instinct is to suggest that you remove the second sentence or revise it to something like "The configuration file is delivered to a host as part of a machine config."